### PR TITLE
VAGOV-2507: Install and Configure Paragraphs Browser Module

### PR DESCRIPTION
**Pull request by acabouet**
Installs, enables, and configures the paragraphs browser module: https://www.drupal.org/project/paragraphs_browser

### CODE REVIEW

**Comment by ElijahLynn** [comment]
@acabouet can you fix this conflict.

**Comment by acabouet** [comment]
rebuild

File: config/sync/paragraphs.paragraphs_type.starred_horizontal_rule.yml [hunk 1]
```diff
@@ -2,10 +2,13 @@ uuid: 90d711dc-f81c-45b1-bc67-304cf1fb917b
 langcode: en
 status: true
 dependencies:
-  content:
```
Comment (beeyayjay) [inline config/sync/paragraphs.paragraphs_type.starred_horizontal_rule.yml]: I don't think we're using starred hr paragraphs, so this yaml shouldn't be included, but double-check with @kevwalsh

**Comment by andyhawks** [comment]
rebuild

**Comment by acabouet** [comment]
rebuild

**Comment by andyhawks** [comment]
rebuild

**Comment by andyhawks** [comment]
rebuild

**Comment by andyhawks** [comment]
rebuild

**Comment by andyhawks** [comment]
rebuild

**Comment by andyhawks** [comment]
rebuild

**Comment by mbenziane** [comment]
rebuild

**Comment by ElijahLynn** [comment]
I am getting failure to apply patches with `lando composer install` on this branch. The latest version might have the patches applied now. 


image

**Comment by ElijahLynn** [comment]
Seems like a `composer update` might have been run globally against core and all modules, which may have updated graphql module instead of just adding the paragraphs browser as intended.

File: composer.lock [hunk 2]
```diff
@@ -4094,62 +4094,6 @@
                 "source": "http://cgit.drupalcode.org/environment_indicator"
             }
         },
-        {
-            "name": "drupal/external_data_source",
```
Comment (andyhawks) [inline composer.lock]: @acabouet is the removal of this intentional?

**Pull request merged**